### ISSUE TITLE
Emulator is not working with Daily limit exceeded

### ISSUE THREAD

**Issue by peakpado** [Scarygami/mirror-api#25]
Hi, tried to connect to glass emulator, but sign in doesn't seem working. Firebug shows the last request failed with Daily Limit.
[
 {
  "error": {
   "code": 403,
   "message": "Daily Limit for Unauthenticated Use Exceeded. Continued use requires signup.",
   "data": [
    {
     "domain": "usageLimits",
     "reason": "dailyLimitExceededUnreg",
     "message": "Daily Limit for Unauthenticated Use Exceeded. Continued use requires signup.",
     "extendedHelp": "https://code.google.com/apis/console"
    }
   ]
  },
  "id": "gapiRpc"
 }
]

**Comment by Scarygami** [comment]
Have you configured your Google Developers Console project and client_secrets correctly as per Readme?

**Comment by peakpado** [comment]
Yes, absolutely, checked a few times, client id, secret and server api key.

**Comment by Scarygami** [comment]
Hmmm, weird, have never run in this problem an I have the exact same version as this repository hosted. Which URL of the emulator are you accessing? Depending on that you might also have to enable the Mirror API in services in the Developers Console.
